client/sammantha.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sammantha Sasenick</title>
+    <!-- Semantic UI -->
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+    <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.js"></script>
+    
+    <!-- My CSS -->
+    <link href="/css/sammantha.css" type="text/css" rel="stylesheet" />
+
+</head>
+<body>
+<div class='container center'>
+    <h1 class="ui center aligned header" id='greeting'>Hey! Nice to meet you.</h1>
+
+    <div class="ui centered cards">
+        <div class="card">
+            <div class="image">
+                <img src="/images/sammantha.jpg">
+            </div>
+            <div class="content">
+                <div class="header">Sammantha</div>
+                <div class="meta">
+                    <a>Coworker</a>
+                </div>
+                <div class="description">
+                    I recently graduated the Coding bootcamp at Vanderbilt University. I have a background in Hospitality, and a Bachelors of Science in Psychology & Neuroscience. In my free time I enjoy cooking, hiking, practicing yoga, and watching netflix with my cat Bobbie.
```

Comment:
This should really be in a `<p>` to keep things semantic :grinning:

Reply:
Got it, sorry! I just pushed the changes.

Reply:
No need to apologize, I'm a very harsh PR reviewer. :)